compiler/rustc_middle/src/ty/c_reader_cache_key.rs

```diff
@@ -0,0 +1,9 @@
+use rustc_hir::def_id::CrateNum;
+
+/// Contains information needed to resolve types and (in the future) look up
+/// the types of AST nodes.
+#[derive(Copy, Clone, PartialEq, Eq, Hash)]
+pub struct CReaderCacheKey {
```

Comment:
Maybe this should be in something other than `ty` -- `rustc_middle::cache`?? idk